src/pages/IstioConfigDetails/IstioObjectDetails/VirtualServiceOverview.tsx

```diff
@@ -66,7 +83,15 @@ class VirtualServiceOverview extends React.Component<VirtualServiceProps> {
       childrenList.push(
         <li key={'gateway_' + host.service + '_' + j}>
           {host.service === 'mesh' || !isValid ? (
```

Comment:
When it is mesh or not valid can be separated, because this causes existing or wrong gateways to be shown as invalid and mesh in the same time when one of gateways is invalid:
![Screenshot from 2021-05-25 07-55-35](https://user-images.githubusercontent.com/604313/119446600-fca8d380-bd2e-11eb-9cc9-6e43307f8c23.png)

Reply:
@hhovsepy Thanks for testing this issue, I made son changes to the code to fix this situation:

![gateway_ok](https://user-images.githubusercontent.com/1286393/119678591-078a6380-be16-11eb-8b33-889ea4a179f0.png)
![gateway_ok_mesh](https://user-images.githubusercontent.com/1286393/119678613-0c4f1780-be16-11eb-89b3-93d8702d4852.png)
![gateway_error](https://user-images.githubusercontent.com/1286393/119678630-0f4a0800-be16-11eb-855b-0ac0ef49986d.png)
![gateway_error_mesh](https://user-images.githubusercontent.com/1286393/119678649-12dd8f00-be16-11eb-989e-95a9813b7948.png)